code/webauthn/contracts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "zksync-hardhat-template",
+  "description": "A template for ZKsync smart contracts development with Hardhat",
+  "private": true,
+  "author": "Matter Labs",
+  "license": "MIT",
+  "repository": "https://github.com/matter-labs/zksync-hardhat-template.git",
+  "scripts": {
+    "deploy": "hardhat deploy-zksync --script deploy.ts",
+    "deploy:NFT": "hardhat deploy-zksync --script deployMyNFT.ts",
+    "deploy:paymaster": "hardhat deploy-zksync --script deployPaymaster.ts",
+    "transfer": "hardhat deploy-zksync --script deployAndTransfer.ts",
+    "register": "hardhat deploy-zksync --script registerR1Owner.ts",
+    "compile": "hardhat compile",
+    "clean": "hardhat clean",
+    "test": "hardhat test --network hardhat"
+  },
+  "devDependencies": {
+    "@matterlabs/hardhat-zksync": "^1.1.0",
+    "@matterlabs/hardhat-zksync-deploy": "^1.3.0",
```

Comment:
```suggestion
```

Reply:
I tried and this doesn't work for me. There is always an issue creating an account. Does it work for you?